kafka/include/etc/confluent/docker/configure

```diff
@@ -84,7 +84,9 @@ then
 fi
 
 # Set if ADVERTISED_LISTENERS has SSL:// or SASL_SSL:// endpoints.
-if [[ -n "${KAFKA_ADVERTISED_LISTENERS-}" ]] && [[ $KAFKA_ADVERTISED_LISTENERS == *"SSL://"* ]]
+# Or if the LISTENER_SECURITY_PROTOCOL_MAP includes SSL
+if ([[ -n "${KAFKA_ADVERTISED_LISTENERS-}" ]] && [[ $KAFKA_ADVERTISED_LISTENERS == *"SSL://"* ]]) ||
```

Comment:
does this work with lower case?

Reply:
No. Is that something that people would do? I don't think the current code covers that case either.